### DISCUSSION TITLE
[task-tester] Store CI configuration in the INGInious repo

### DISCUSSION THREAD

**Discussion by nrybowski** [INGInious/INGInious#896] [Q&A] [answered]
Currently, the GitHub CI configuration is stored in the .github repo of the UCL-INGI organization (see https://github.com/UCL-INGI/.github/pull/1) to ease the deployment of the task-tester in course repositories.

This can be confusing for external users of the platform and, in fine, the configuration belongs to the INGInious project. Hence, should we not also store the task-tester CI configuration in the main repository? In that case, how do we handle the duplication between both repositories?

**Comment by anthonygego** [comment, marked as answer]
As we have discussed informally the other day, I think that the main issue is way much larger than just `.github` common repo as the project is using more and more subrepos next to the department unrelated repositories (or related but not of common interest), making confusion for external people, making us prefix the repositories associated to the project, ending with super long repository names, while sometimes other department people still add the project name in their repo without us being able to control that. 

In addition to all of these pains, it has always made managing people in the UCL-INGI organization quite tricky and would make it even more complex in case we would add R/W access to external people, which could actually be something legitimate (BTW, @GuillaumeDerval actually is, now) .

For several years, we've been in posession of the INGIniousOrg organization on Github, initially to store some private repositories to develop cloud services around the project. We never used it for public repos as the name `inginious` was already used by another user and wouldn't have eased finding the project. Today as I was passing by, reading this, I realized the Github name INGInious was free and took the opportunity to rename the organization. 

So we actually own the https://github.com/INGInious URL now ! :partying_face: 

For me, all the conditions are now met to start migrating the INGInious repositories to their own organization and we should work on that as soon as we're ready to.

**Reply by GuillaumeDerval** [reply]
🥳
